spec/hydra/works/models/concerns/generic_file/characterization_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+# Mocking out the Hydra::Works::GenericFile sufficiently to add original_file
```

Comment:
Here's a (slightly more complicated than you need) example of how to make this work.

First, write your function and an rspec config include in e.g. `spec/support/fedora_mocks.rb`. Example: https://github.com/projecthydra/sufia/blob/a064489d3cf4d228abf978d7126644e2c653aa5e/spec/support/proxies.rb#L2-L13

Then require it in your spec_helper. (This example happens in two steps to combine all feature spec-related helpers together, but you wouldn't need both steps: https://github.com/projecthydra/sufia/blob/a064489d3cf4d228abf978d7126644e2c653aa5e/spec/spec_helper.rb#L23 and https://github.com/projecthydra/sufia/blob/a064489d3cf4d228abf978d7126644e2c653aa5e/spec/support/features.rb#L5 You could just `require 'support/fedora_mocks'` in spec_helper.)

Then call your neatly packaged method: https://github.com/projecthydra/sufia/blob/a064489d3cf4d228abf978d7126644e2c653aa5e/spec/features/proxy_spec.rb#L14